javascript/javascript3/week1/Gif/scripts.js

```diff
@@ -0,0 +1,23 @@
+const APIKEY = "HE28dRYots7IvgGuuRC1NR8AEYbHqqyK";
+const searchInput = document.getElementById("search-input");
+const numberInput = document.getElementById("number-input");
+const button = document.getElementById("button");
+const results = document.getElementById("results");
+
+button.addEventListener("click", (btn)=>{
+    btn.preventDefault();
+    fetch(`https://api.giphy.com/v1/gifs/search?api_key=${APIKEY}&limit=${numberInput.value}&lang=en&q=${searchInput.value}`)
+    .then(response =>{
+        return response.json();
+    }).then(JSONData =>{
+        for(let i = 0; i<numberInput.value; i++){
```

Comment:
What would happen if the API only returned 2 images when you asked for 3?
I think it would be safer to loop over the length of `JSONData.data` instead!